lib/graphql.ex

```diff
@@ -180,4 +184,23 @@ defmodule BridgeEx.Graphql do
       end
     end
   end
+
+  defmacro preload_query_from_file(name, opts \\ []) when is_atom(name) do
```

Comment:
I thougth about it, but it seems to give too many responsibilities to the main "using" macro.
Maybe it could live in a separate module, since it is an optional feature, so it could be something like this:
```
use Graphql.PreloadedQueries,
  my_query: "query.graphql",
  my_mutation: "mutation.graphql"
```

But in this case I can't see a natural way to use a default path.
With a classic macro function we can easly define other options:
```
preload_query_from_file :my_query
preload_query_from_file :my_query, path: "banana"
```

Reply:
I agree that having a macro that has side-effects is not so "explicit", but I also agree that overloading the `use BridgeEx.Graphql` macro is not ideal either because it's becoming very big.

I kinda like the idea of having a separate extension like

```elixir
use BridgeEx.Extensions.PreloadedQueries,
  # This is expected to be at `./hello_query.graphql`
  :hello_query,
  # These are explicit paths that will be taken as is
  get_user: "./user/queries/get_user.graphql"
  delete_user: "./user/mutations/delete_user.graphql"
```

The option of just letting the user provide a name without a path could maybe be removed...? I don't know how common it is to have a query in the directory local to the bridge.

Let me know what you think!

Reply:
Actually we could remove the default path. Honestly I don't think it is a "must to have". So ok for me 👍 

About the name... what about `BridgeEx.Extensions.ExternalResources`? It is more generic and consistent with the Elixir naming.

Reply:
Yes it makes sense, since many different resources could be preloaded 👍